BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -347,3 +335,69 @@ extension UIAccessibilityTraits {
     }
 }
 
+
+extension AccessibilityElement {
+    /// Used to provide additional functionality to assistive technologies beyond your accessible UI.
+    public struct CustomAction: Equatable, Hashable {
+        public typealias OnActivation = () -> Bool
+
+        /// A localized name that discribes the action.
+        public var name: String
+        /// An image representing the action to be shown with some assistive technologies such as Switch Control.
+        public var image: UIImage?
+        /// A Callback for when the action is activated. This should return a `bool` indicating success or failure of the action.
+        public var onActivation: OnActivation
+
+        public init(name: String, image: UIImage? = nil, onActivation: @escaping OnActivation) {
+            self.name = name
+            self.image = image
+            self.onActivation = onActivation
+        }
+
+        public static func == (lhs: AccessibilityElement.CustomAction, rhs: AccessibilityElement.CustomAction) -> Bool {
+            // Disregard onActivation for equatablity pruposes.
+            lhs.name == rhs.name && lhs.image == rhs.image
+        }
+
+        public func hash(into hasher: inout Hasher) {
+            // Disregard onActivation for hash pruposes.
+            hasher.combine(name)
+            hasher.combine(image)
+        }
+    }
+}
+
+extension AccessibilityElement {
+    public struct CustomContent {
+        public enum Importance: Equatable { case regular, high }
```

Comment:
Can you document this a bit? Also, maybe split onto multiple lines like regular enums